spec/acceptance/suites/default/10_one_node_spec.rb

```diff
@@ -20,8 +20,8 @@
   context 'basic docker usage' do
     hosts.each do |host|
       it 'should apply with no errors' do
-        on(host, "sed -i 's/enforce_for_root//g' /etc/pam.d/*")
-        on(host, 'echo "root:password" | chpasswd --crypt-method SHA256')
+        #on(host, "sed -i 's/enforce_for_root//g' /etc/pam.d/*")
```

Comment:
@jeefberkey If you're going to comment these out and leave them, there should probably be an explanation comment.

Reply:
I will revert it, I needed a new commit to poke travis.